tests/test_pytomorrowio.py

```diff
@@ -182,6 +184,42 @@ async def test_timelines_daily_good(aiohttp_client):
         assert set(values) == set(available_fields)
 
 
+async def test_timelines_daily_greater_than_max_fields_good(aiohttp_client):
+    fields = []
+    for field in CORE_FIELDS:
+        fields.append(f"{field}Min")
+        fields.append(f"{field}Max")
+        fields.append(field)
```

Comment:
In my testing, `field` has the same value as `fieldMax`.

Reply:
hmm I thought it was the same as `fieldAvg`. Will check again, but the main thing here was to generate more than 55 fields so we could test the merge logic